tests/test_energy_overflows.py

```diff
@@ -18,6 +18,10 @@ def verify_energies(a, b):
         assert np.isnan(a) and np.isnan(b)
 
 
+def fixed_overflowed(a):
+    return a <= np.iinfo(np.int64).min or a >= np.iinfo(np.int64).max
```

Comment:
Not sure I understand why this uses `np.int64`, when I thought we were moving to `int128`? Docstring here seems useful.

Reply:
Sorry, premature comment. I think I understand this now. (I think docstring would still be useful)

Reply:
Added comment point the user to read the C++ docs : [2cc4964](https://github.com/proteneer/timemachine/pull/1090/commits/2cc4964310da1dbc9dac9d2e7b08c1b323f2e5ad)